dao/OrderRepository.go

```diff
@@ -0,0 +1,118 @@
+package dao
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+)
+
+type OrderPostgres struct {
+	db *sql.DB
+}
+
+func NewOrderPostgres(db *sql.DB) *OrderPostgres {
+	return &OrderPostgres{db: db}
+}
+
+type Order struct {
+	IdDeliveryService int    `json:"delivery_service_id,omitempty"`
+	Id           int    `json:"id"`
+	IdCourier         int    `json:"courier_id,omitempty"`
+	DeliveryTime      string `json:"delivery_time,omitempty"`
+	CustomerAddress   string `json:"customer_address,omitempty"`
+	Status            string `json:"status"`
+	OrderDate         string `json:"order_date"`
+	RestaurantAddress string `json:"restaurant_address"`
+	Picked            bool   `json:"picked"`
+}
+
+type DetailedOrder struct {
+	IdDeliveryService  int    `json:"delivery_service_id,omitempty"`
+	Id            int    `json:"id"`
+	IdCourier          int    `json:"courier_id,omitempty"`
+	DeliveryTime       string `json:"delivery_time,omitempty"`
+	CustomerAddress    string `json:"customer_address,omitempty"`
+	Status             string `json:"status,omitempty"`
+	OrderDate          string `json:"order_date,omitempty"`
+	RestaurantAddress  string `json:"restaurant_address,omitempty"`
+	Picked             bool   `json:"picked,omitempty"`
+	CourierName        string `json:"name,omitempty"`
+	CourierPhoneNumber string `json:"phone_number,omitempty"`
+}
+
+func (r *OrderPostgres) AssigningOrderToCourier_InDB(order Order) error {
+	transaction, err := r.db.Begin()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Println("connected to db")
+
+	defer transaction.Commit()
+	s := fmt.Sprintf("UPDATE delivery SET courier_id = %d WHERE id = %d", order.IdCourier, order.Id)
```

Comment:
why don't you want to have it like a query with params instead of Sprintf? Of course integers couldn't cause SQL injection, but just for consistency or if you decide to update more fields in the future.